src/main/java/net/minecraftforge/registries/RegistryManager.java

```diff
@@ -161,10 +162,21 @@ <V extends IForgeRegistryEntry<V>> ForgeRegistry<V> createRegistry(ResourceLocat
         return getRegistry(name);
     }
 
-    @SuppressWarnings("unchecked")
     static <V extends IForgeRegistryEntry<V>> void registerToRootRegistry(ForgeRegistry<V> forgeReg)
     {
-        WritableRegistry<Registry<V>> registry = (WritableRegistry<Registry<V>>) Registry.REGISTRY;
+        injectForgeRegistry(forgeReg, Registry.REGISTRY);
+    }
+
```

Comment:
Formatting: there should be at most one blank line between lines with content.
```suggestion
```

Reply:
Fixed in next push